src/components/dialogs/ConfimRemoveDialog.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="confirm-remove-dialog">
+    <v-card>
+      <v-card-title id="dialog-title">
+        Remove Person or Corporation / Firm
+      </v-card-title>
+
+      <v-card-text id="dialog-text">
+        <p class="genErr">Remove this Person or Corporation or Firm from your Company?</p>
+      </v-card-text>
+
+      <v-divider class="my-0"></v-divider>
+
+      <v-card-actions>
+        <v-spacer></v-spacer>
+        <v-btn id="dialog-remove-btn" color="primary" text @click="confirm()">Remove</v-btn>
+        <v-btn id="dialog-cancel-btn" text @click="exit()">Cancel</v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+import { Component, Vue, Prop, Emit } from 'vue-property-decorator'
+
+@Component
+export default class ConfirmRemoveDialog extends Vue {
```

Comment:
Would it make more sense to use the generic Confirm Dialog here (as this is such a simple component)?

Reply:
I think anything is possible! This might require some more time than we have at this present moment. Once again i would request we extend this refactor until we time to revisit this.

Reply:
OK, don't worry about it.